src/backend/marsha/settings.py

```diff
@@ -154,6 +154,30 @@ class Base(Configuration):
     AUTH_USER_MODEL = "core.User"
 
     ASGI_APPLICATION = "marsha.asgi.application"
+    
+    
+    # For sentinels:
+    # CHANNEL_LAYERS = {
+    #     'default': {
+    #         'BACKEND': 'channels_redis.core.RedisChannelLayer',
+    #         'CONFIG': {
+    #             "hosts": [{
+    #                 "sentinels": [(SENTINEL_HOST, SENTINEL_PORT)]m
+    #                 "master_name": SENTINEL_MASTER,
+    #             }],
+    #         },
+    #     },
+    # }
+    CHANNEL_LAYERS = {
+        "default": {
+            "BACKEND": "channels_redis.core.RedisChannelLayer",
+            "CONFIG": {
+                "hosts": values.ListValue(
+                    [("redis", 6379)], environ_name="REDIS_HOST", environ_prefix=None
```

Comment:
I don't get this line, how does environ_name works ?
Should we add something to env.d/development.dist ?

Also I saw [In-Memory Channel Layer](https://channels.readthedocs.io/en/stable/topics/channel_layers.html#in-memory-channel-layer) maybe it would be useful for testing settings ?